docs/resources/vault_secrets_integration_postgres.md

```diff
@@ -0,0 +1,57 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_vault_secrets_integration_postgres Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Vault Secrets Twilio integration resource manages a Twilio integration.
+---
+
+# hcp_vault_secrets_integration_postgres (Resource)
+
+The Vault Secrets Twilio integration resource manages a Twilio integration.
+
+## Example Usage
+
+```terraform
+resource "hcp_vault_secrets_integration_confluent" "example" {
```

Comment:
Should this be postgres instead of confluent?

Reply:
yeah... that's weird.

the docs are all auto-generated